qemu/qmp.go

```diff
@@ -946,11 +946,13 @@ func (q *QMP) ExecutePCIVFIOMediatedDeviceAdd(ctx context.Context, devID, sysfsd
 		"id":       devID,
 		"driver":   "vfio-pci",
 		"sysfsdev": sysfsdev,
-		"addr":     addr,
 	}
 	if bus != "" {
 		args["bus"] = bus
 	}
+	if addr != "" {
```

Comment:
We should also update the documentation.  Currently, it simply says bus is optional.  But this commit makes addr optional as well.  The comment in the commit message is good.  Could we add it to the documentation.  So have something like,

"Both bus and addr are optional.  If they are both set to empty the system will pick up
an empty slot on root bus."

Reply:
ok, I will do it right now.